scripts/minikube.sh

```diff
@@ -51,11 +51,21 @@ function detect_minikube() {
 # install minikube
 function install_minikube() {
     if type "${minikube}" >/dev/null 2>&1; then
-        local mk_version version
+        local mk_version version mku_version latest_version ret
         read -ra mk_version <<<"$(${minikube} version)"
         version=${mk_version[2]}
         echo "minikube already installed with ${version}"
-        return
+        mku_version=$(${minikube} update-check 2> /dev/null || ret=${?})
```

Comment:
shellcheck (part of `make-containerized-test`) agrees that changing `ret` in a sub-shell is not working as you intend.

Reply:
I had tested it enabling `set -x` with various conditions and it worked for me and logically it looks fine too :-)
Any have, now that it bothers shellcheck, I had made some changes and updated the PR. Please take a look.

Thanks @nixpanic